### DISCUSSION TITLE
Does it supports Asus Rog Strix g16 (2023)?

### DISCUSSION THREAD

**Discussion by MemoSainz** [seerge/g-helper#2617] [Q&A]
Is this software supporting this laptop? Sorry, I see that it supports Zephyrus G16 and also Scar g16. But I don't see if it supports Rog Strix. Perhaps I'm wrong or didn't made a good research in the FAQ nor the discussion. But I wish to know. 

Please don't ban me for a silly question, as I want to love this software and use it and have a good opinion o it. Because I hate the crap and bloat from Asus.

**Comment by seerge** [comment]
@MemoSainz hello, it does support all modern Asus laptops.

**Reply by MemoSainz** [reply]
thank you!
I also edited the year, because it was 2023 and not 2013.

will it support mine, right?

**Reply by Kikimahe** [reply]
Yes, "support all modern Asus laptops".

In a few seconds you can answer this question yourself. It's only 1 *.exe file.
- Donloawd Ghelper
- Run it
- Test the features.

If you like it, uninstall Armory crate with [the ASUS uninstaller tool](https://dlcdnets.asus.com/pub/ASUS/mb/14Utilities/Armoury_Crate_Uninstall_Tool.zip?model=armoury%20crate).
